tests/helpers.py

```diff
@@ -90,3 +90,11 @@ def send(self, *args):
 
 
 _MockResponse = namedtuple('MockResponse', ['status_code', 'text', 'headers'])
+
+
+def assert_raises(exception_class, action):
+  try:
+    action()
+    raise AssertionError, "Should have raised an %s" % exception_class
```

Comment:
I would have expected this to assert that the func passed must raise the exception class provided, instead of being chainable like this.